content/docs/setup/kubernetes/multicluster-install/gateways/index.md

```diff
@@ -0,0 +1,238 @@
+    --from-file=path/to/intermediate/cert-chain.pem
+$ kubectl apply -f $HOME/istio.yaml
+{{< /text >}}
+
+For further details and customization options, refer to the [Installation
+with Helm](/docs/setup/kubernetes/helm-install/) page.
+
+> The installation steps above assume that you have generated an
+> intermediate CA for each cluster that will be part of the mesh.
+
+## Enable mTLS globally
+
+Cross cluster communication via the Istio gateway requires mTLS to be
+enabled for all services in each cluster. The global authentication policy
+must be set to permissive or stricter. In addition, destination rule must
+be set to originate mutual TLS connection from every pod. For example,
+apply the following global configuration in each cluster:
+
+{{< text yaml >}}
+apiVersion: authentication.istio.io/v1alpha1
+kind: MeshPolicy
+metadata:
+  name: default
+spec:
+  peers:
+  - mtls:
+      mode: PERMISSIVE
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: default
+spec:
+  host: "*.local"
+  trafficPolicy:
+    tls:
+      mode: ISTIO_MUTUAL
+{{< /text >}}
+
+## Replicate services and namespaces across clusters
+
+In order for pods in a cluster to be able to resolve DNS names of services
+in remote clusters, we need to replicate the namespace and service
+declaration from one cluster to other clusters. In addition, to route
+traffic to a remote service via the Istio Gateway, an Istio Service Entry
+configuration should be added for each replicated service. For example, the
+diagram above depicts two services `foo.ns1` in `cluster1` and `bar.ns2` in
+`cluster2`. Lets say their respective declarations are as follows:
+
+In `cluster1` (file: `cluster1-svc.yaml`)
+
+{{< text yaml >}}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ns1
+  labels:
+    istio-injection: enabled
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: foo
+  labels:
+    app: foo
+  namespace: ns1
+spec:
+  ports:
+  - port: 80
+    targetPort: 80
+    name: http
+  selector:
+    app: foo
+{{< /text >}}
+
+In `cluster2` (file: `cluster2-svc.yaml`)
+
+{{< text yaml >}}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ns2
+  labels:
+    istio-injection: enabled
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: bar
+  labels:
+    app: bar
+  namespace: ns2
+spec:
+  ports:
+  - port: 80
+    targetPort: 80
+    name: http
+  - port: 9090
+    targetPort: 9090
+    name: tcp
+  selector:
+    app: bar
+{{< /text >}}
+
+The namespaces and services need to be replicated in each cluster in order
+to enable DNS resolution and connectivity. So, apply the configuration in `cluster1-svc.yaml` to
+`cluster2` and the configuration in `cluster2-svc.yaml` to `cluster1`.
+
+## Route traffic for remote services to remote cluster gateway
+
+Creating a service object for a remote service enables DNS resolution for
+remote services. The DNS is solely for the application convenience. Istio
```

Comment:
"The DNS is solely for the application emote services. The DNS is solely for the application convenience" this might read a bit better with something like "DNS resolution is for application convenience or requirements"